application.go

```diff
@@ -99,6 +98,8 @@ type Application struct {
 	LastTaskFailure       *LastTaskFailure        `json:"lastTaskFailure,omitempty"`
 	Fetch                 *[]Fetch                `json:"fetch,omitempty"`
 	IPAddressPerTask      *IPAddressPerTask       `json:"ipAddress,omitempty"`
+	Env                   map[string]string       `json:"env,omitempty"`
```

Comment:
Do we actually need the `json` spec now? IIUC, we are only marshaling and unmarshaling through separate and distinct structs by now.